AdornUI-Stylesheets/badges.css

```diff
@@ -0,0 +1,105 @@
+h1 {
+  font-size: 32px;
```

Comment:
For bigger font sizes , use relative units. No need to change for smaller font sizes.